src/pages/workspaces/workspace/workflows/WorkflowView.js

```diff
@@ -445,14 +444,21 @@ const WorkflowView = _.flow(
         entityMetadata,
         savedInputsOutputs: inputsOutputs,
         modifiedInputsOutputs: inputsOutputs,
-        snapshotIds,
         errors: isRedacted ? { inputs: {}, outputs: {} } : augmentErrors(validationResponse),
         entitySelectionModel: this.resetSelectionModel(modifiedConfig.rootEntityType, readSelection ? selection.entities : {}),
         workspaceAttributes: _.flow(
           _.without(['description']),
           _.remove(s => s.includes(':'))
         )(_.keys(attributes))
       })
+
+      if (sourceRepo === 'agora') {
+        const methods = await Methods.list({ namespace: methodNamespace, name: methodName })
+        const snapshotIds = _.map(m => _.pick('snapshotId', m).snapshotId, methods)
```

Comment:
nit: While we're changing this code, this can be simplified to `_.map('snapshotId', methods)`